dump/config/bufr2ioda_bufr_backend_satwnd_amv_goes.yaml

```diff
@@ -0,0 +1,58 @@
+time window:
+  begin: "2018-04-14T21:00:00Z"
```

Comment:
just noting that we will want to template these if they are not in a fixed 'test' directory

Reply:
I missed this comment. 
This configuration file is for CTEST only.  So, I think a pre-defined time window should be OK.